src/stream/src/executor/monitor/streaming_stats.rs

```diff
@@ -1206,6 +1209,14 @@ impl StreamingMetrics {
         .unwrap()
         .relabel_debug_1(level);
 
+        let gap_fill_generated_rows_count = register_guarded_int_counter_vec_with_registry!(
+            "stream_gap_fill_generated_rows_count",
+            "Total number of rows generated by gap fill executor",
+            &["actor_id", "fragment_id"],
+            registry
+        )
+        .unwrap();
```

Comment:
We should `relabel_debug_1` here. In general we don't expose per-actor metrics in production due to high cardinality. Although gapfill has singleton distribution, we should retain this convention here, in case there are any changes in the future, and to avoid any confusion.